tests/pipelines/stable_unclip/test_stable_unclip.py

```diff
@@ -0,0 +1,219 @@
+import gc
+import unittest
+
+import torch
+from transformers import CLIPTextConfig, CLIPTextModel, CLIPTextModelWithProjection, CLIPTokenizer
+
+from diffusers import (
+    AutoencoderKL,
+    DDIMScheduler,
+    DDPMScheduler,
+    NoiseAugmentor,
+    PriorTransformer,
+    StableUnCLIPPipeline,
+    UNet2DConditionModel,
+)
+from diffusers.utils.testing_utils import load_numpy, require_torch_gpu, slow, torch_device
+
+from ...test_pipelines_common import PipelineTesterMixin, assert_mean_pixel_difference
+
+
+class StableUnCLIPPipelineFastTests(PipelineTesterMixin, unittest.TestCase):
+    pipeline_class = StableUnCLIPPipeline
+
+    # TODO(will) Expected attn_bias.stride(1) == 0 to be true, but got false
+    test_xformers_attention = False
```

Comment:
xformers doesn't work yet?

Reply:
Yeah we get that error `Expected attn_bias.stride(1) == 0 to be true, but got false`.

I was going to focus on that after merging the main pipelines